README.md

```diff
@@ -91,32 +102,26 @@ each installation:
    for API, Ingress, and DNS access.
 1. The BMC IPMI addresses and credentials for the 3 control plane
    machines.
-1. Optionally, a Network Time Protocol (NTP) server where the default
-   public server is not accessible
+1. If detected that the provsioning host is not sync with a time source, configure the 25G switch as a source via the Storage DHCP and provide an optional script to set a source for the switch. 
 
 ## Provisioning Host Setup
 
 The provisioning host must be a RHEL-8 machine.
 
-Make a copy of `config_example.sh` and set the required variables in
+### For a host not installed using the ISO:
+In the OpenShift subdirectory, Make a copy of `config_example.sh` and set the required variables in
 there.
 
-To install some required packages and the `oc` client:
+To install some required packages configure libvirt, and prepare the 
```

Comment:
add ticks to libvirt. Change to `libvirt` --- FYI - Libvirt will be installed by ISO so not sure how you want to rewrite all this portion in general.

Reply:
This is in a section with the header "For a host not installed using the ISO:"